.github/workflows/deploy_infrastructure.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
     - name: Install dependencies
       run: npm ci --legacy-peer-deps
     - name: Deploy Dev
-      run: DEV_CERT_ARN=${{ vars.DEV_CERT_ARN }} BASE_DOMAIN=${{ vars.BASE_DOMAIN }} npm run deploy -- --require-approval never
+      run: ADMIN_EMAIL=jack@gnar.dog DEV_CERT_ARN=${{ vars.DEV_CERT_ARN }} BASE_DOMAIN=${{ vars.BASE_DOMAIN }} npm run deploy -- --require-approval never
```

Comment:
can this be `vars.ADMIN_EMAIL` like the others? Otherwise we need a PR to update the admin email, right?

Reply:
It will be! I just want to make sure it works before asking Tashrif to set the variable to whoever from MGB will be testing the feature. So I'll test and send a follow-up PR with the GA variable.